Week1/homework/world-queries.txt

```diff
@@ -1,40 +1,362 @@
+| Costa Rica                       |
+| Cuba                             |
+| Cayman Islands                   |
+| Dominica                         |
+| Dominican Republic               |
+| Guadeloupe                       |
+| Grenada                          |
+| Greenland                        |
+| Guatemala                        |
+| Honduras                         |
+| Haiti                            |
+| Jamaica                          |
+| Saint Kitts and Nevis            |
+| Saint Lucia                      |
+| Mexico                           |
+| Montserrat                       |
+| Martinique                       |
+| Nicaragua                        |
+| Panama                           |
+| Puerto Rico                      |
+| El Salvador                      |
+| Saint Pierre and Miquelon        |
+| Turks and Caicos Islands         |
+| Trinidad and Tobago              |
+| United States                    |
+| Saint Vincent and the Grenadines |
+| Virgin Islands, British          |
+| Virgin Islands, U.S.             |
++----------------------------------+
+37 rows in set (0.00 sec)
+
+---------------------------------------------------------------------------------
+4- What's the top 10 countries by SurfaceArea ?
+SELECT country.name, country.SurfaceArea FROM country ORDER BY SurfaceArea DESC LIMIT 10;
++--------------------+-------------+
+| name               | SurfaceArea |
++--------------------+-------------+
+| Russian Federation | 17075400.00 |
+| Antarctica         | 13120000.00 |
+| Canada             |  9970610.00 |
+| China              |  9572900.00 |
+| United States      |  9363520.00 |
+| Brazil             |  8547403.00 |
+| Australia          |  7741220.00 |
+| India              |  3287263.00 |
+| Argentina          |  2780400.00 |
+| Kazakstan          |  2724900.00 |
++--------------------+-------------+
+10 rows in set (0.00 sec)
+
+---------------------------------------------------------------------------------
+5- What's the top 10 most populated cities?
+ SELECT city.name, city.population FROM city ORDER BY population DESC LIMIT 10;
++-----------------------+------------+
+| name                  | population |
++-----------------------+------------+
+| Mumbai (Bombay)       |   10500000 |
+| Seoul                 |    9981619 |
+| Sï¿½o Paulo           |    9968485 |
+| Shanghai              |    9696300 |
+| Jakarta               |    9604900 |
+| Karachi               |    9269265 |
+| Istanbul              |    8787958 |
+| Ciudad de Mï¿½xico    |    8591309 |
+| Moscow                |    8389200 |
+| New York              |    8008278 |
++-----------------------+------------+
+10 rows in set (0.00 sec)
+
+---------------------------------------------------------------------------------
+6- Select the top 3 countries by population that their names start with the letter ‘P’?
+ SELECT country.name, country.population FROM country WHERE country.name LIKE 'p%' LIMIT 3;
```

Comment:
To get the top 3 countries by population you need to use the `order by` clause too:

`SELECT country.name, country.population FROM country WHERE country.name LIKE 'p%' ORDER BY population DESC LIMIT 3;`